pkg/parser/parser.go

```diff
@@ -17,7 +17,7 @@ type kindParser interface {
 	SupportedExtensions() []string
 	SupportedTypes() map[string]bool
 	Parse(filePath string, fileContent []byte) ([]model.Document, []int, error)
-	Resolve(fileContent []byte, filename string) ([]byte, error)
+	Resolve(fileContent []byte, filename string, _ bool) ([]byte, error)
```

Comment:
since this is the declaration of the method you should keep the variable name as it provides some insight on what it is